torch/functional.py

```diff
@@ -13,6 +13,7 @@
 
 Tensor = torch.Tensor
 from torch import _VF
+from torch._C import _linalg  # type: ignore[attr-defined]
 
 __all__ = [
```

Comment:
Removing `tensordot` from `__all__` would decouple `torch.tensordot` and `torch.functional.tensordot`. Then you'd get it as an alias to `torch._C._linalg.linalg_tensordot` and it should be possible to add a separate docstring in `_torch_docs.py` that would be simply referring to `torch.linalg.tensordot`.

Reply:
I don't think this is doable: if I make `torch.tensordot` an alias to `torch._C._linalg.linalg_tensordot` it would have a different signature and behavior as `torch.linalg.tensordot`, because the latter has a python implementation.